core/src/test/java/io/github/tcdl/RequesterTest.java

```diff
@@ -152,7 +152,8 @@ private Requester initRequesterForResponsesWithTimeout(int numberOfResponses) th
         when(channelManagerMock.findOrCreateProducer(anyString())).thenReturn(producerMock);
 
         MsbContext msbContext = TestUtils.createSimpleMsbContext();
-        msbContext.setChannelManager(channelManagerMock);
+        msbContext = TestUtils.createMsbContext(msbContext.getMsbConfig(), msbContext.getMessageFactory(), 
```

Comment:
Looks tricky. Couldn't we provide an easier way to specify channel manager instance?